<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>821</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Program liczący pierwiastki wielomianu metodą Newtona za pomocą zmiennopozycyjnej oraz przedziałowej arytmetyki.</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="boxArithmeticChoice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Wybierz arytmetykę:</string>
    </property>
    <widget class="QRadioButton" name="radioButtonArithmeticNormal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>zmiennopozycyjna</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonArithmeticInterval">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>przedziałowa</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="boxNChoice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Wprowadź n:</string>
    </property>
    <widget class="QSpinBox" name="spinBoxN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="boxXChoice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Wprowadź x:</string>
    </property>
    <widget class="QLineEdit" name="lineEditX_a">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditX_b">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelX_a">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x.a</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelX_b">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x.b</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="boxMitChoice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Wprowadź maksymalną liczbę iteracji:</string>
    </property>
    <widget class="QSpinBox" name="spinBoxMit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="boxEpsChoice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Wprowadź epsilon:</string>
    </property>
    <widget class="QLineEdit" name="lineEditEps">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="boxVector">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>391</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Wprowadź wektor danych a[]:</string>
    </property>
    <widget class="QPushButton" name="pushButtonAiAdd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Dodaj</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAiRemove">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>510</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuń</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetVector">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>401</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>170</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>57</number>
     </attribute>
     <column/>
     <column/>
    </widget>
    <widget class="QPushButton" name="pushButtonRemoveAll">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>510</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuń wszystkie</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAi_a">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>440</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAi_b">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>470</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelAi_a">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>a_lewy</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAi_b">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>a_prawy</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="boxResults">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Wyniki:</string>
    </property>
    <widget class="QTableWidget" name="tableWidgetResults">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>551</width>
       <height>281</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>6</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>243</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>42</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonCount">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>380</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Licz!</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAuthor">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>600</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Projekt wykonany przez Zbigniewa Andrzejewskiego pod opieką prof. dr hab. Andrzeja Marciniaka</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserInstruction">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>480</y>
      <width>471</width>
      <height>101</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Domyślne wartości n, x, mit i eps, a_i to 0. W każdej chwili można zmienić arytmetykę. a_prawy zawsze musi być większe od a_lewy. Wartości wpisane w polach x.i, eps oraz a_i należy zaakceptować wciskając Enter. W przypadku liczenia arytmetyką zmiennopozycyjną, brane są wartości x.a oraz a_lewy.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
